### ISSUE TITLE
Sparse time zones, difficult to setup (suggestion)

### ISSUE THREAD

**Issue by ddnexus** [Masquerade-Circus/gnome-timezones-extension#10]
Example: if you want to add San Francisco - USA, you are out of luck.

I guess that having all the biggest cities in the world with the current UI would be super messy if not impossible, but what about having at least all the time zones (just by UTC offset, or the list will be longer)?

Then if you can pick the time zone you want, it would be nice to have the possibility to edit it with your own city-label, so the extension could cover all the places in the world with just the time offsets and an editable field.

**Comment by robmsmt** [comment]
I agree, I was going to suggest something similar. I worked with a college in a remote part of a country I was unfamiliar with and since the country itself was not listed, it took some map searches to get their nearest approximate big city that was also listed in this extension.

**Comment by mfuhrmann** [comment]
Same with Gran Canaria. WEST or GMT+1. Can't find it easily.
